src/bundle/Resources/views/fieldtypes/edit/ezdate.html.twig

```diff
@@ -7,8 +7,8 @@
             </svg>
         </button>
     </div>
-    {% set offset =  form.parent.vars.value.value.date|length ? form.parent.vars.value.value.date.offset : null %}
+    {% set contentId = app.request.attributes.get('contentId') %}
     {% set attr = attr|merge({'class': 'ez-data-source__input', 'hidden': 'hidden',
-        'data-timezone-offset': offset}) %}
+        'data-content-id': contentId}) %}
```

Comment:
This doesn't really follow the specification from the description
> There are basically two scenarios: creating new Content with ezdate fieldtype or updating one:

I understand that it works now because Content ID is available during update only, but the driver for the strategy applied in JS should be action type, not the presence of Content ID as the latter one is rather unstable, especially that it relies on a request.

AFAICS the alternative possible out of the box ain't pretty either:
```twig
app.request.attributes.get('_route')
```
which is:
* `ez_content_create_no_draft` for create
* `ez_content_draft_edit` for update

but it still feels unstable 'cause it relies on a `request`.

Regardless of the direction above, I'd see this modeled in a different way:
```suggestion
        'data-action-type': action_type}) %}
```
where `action_type` is one of e.g.: `update` or `create`

Ideally though if this would come frome some strictly defined place, not Content Id presence nor route name based. Not sure if It's worth the effort though.

POV ping @webhdx @Nattfarinn

Reply:
@alongosz done based on `_route` param: https://github.com/ezsystems/ezplatform-admin-ui/pull/1350/commits/fc10e49ffcdec6c96b6adb38b4f9798d27e007ad